docs/example.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE HTML>
```

Comment:
This is just workaround for Read The Docs to redirect `/example` to `/quickstart`. I merged those two pages.